lib/io/hardware_gpio.hpp

```diff
@@ -37,6 +68,14 @@ class HardwareGpio {
 
  private:
   hyped::core::ILogger &log_;
+  const off_t bankAddresses[4] = {0x44e07000, 0x4804c000, 0x481ac000, 0x481ae000};
```

Comment:
Where are these values from? Is it documented somewhere?

Reply:
Taken from old code and also cross referenced from AM335x technical reference manual along with other tutorials.

Reply:
Ok sounds good. Is this documented somewhere so that  someone next year can know where these values came from?

Reply:
Also in general "it was in the old codebase" is not justification. We are writing a new system here with lots of new components and there was lots of flawed legacy code in the 2022 codebase. If you are ever unsure of a value, please either ask someone who may already know (i.e. @SnickeyX) or find the relevant datasheet and document where the values came from.